api/cedar_metadata_templates/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.conf.urls import re_path
+
+from api.cedar_metadata_templates import views
+
+app_name = 'osf'
+
+urlpatterns = [
+    re_path(r'^$', views.CedarMetadataTemplateList.as_view(), name=views.CedarMetadataTemplateList.view_name),
+    re_path(r'^(?P<template_id>[-0-9A-Za-z]+)/$', views.CedarMetadataTemplateDetail.as_view(), name=views.CedarMetadataTemplateDetail.view_name),
```

Comment:
I don't think `_id` has `-` in it. Thus `[0-9A-Za-z]` should work fine.

See  [`bson.ObjectId()`](https://github.com/CenterForOpenScience/osf.io/blob/1722bbd83cf2777d8f936b274df54bdcca384ea9/osf/models/base.py#L47C16-L47C31) in [`ObjectIDMixin`](https://github.com/CenterForOpenScience/osf.io/blob/1722bbd83cf2777d8f936b274df54bdcca384ea9/osf/models/base.py#L249)